site/content/blogs/release-v131.en.md

```diff
@@ -34,6 +34,15 @@ Copilot v1.31 brings big enhancements to help you develop more flexibly and effi
 
 ## NLB enhancements
 
+Copilot brings UDP traffic support with an update to your Network Load Balancer! The protocol your NLB uses is specified by the [nlb.port](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#nlb-port) field.
+```
+nlb:
+  port: 8080/udp
+```
+
+!!!warning
```

Comment:
Maybe an "info" instead of "warning"? Also, maybe it's better to let them know what's the possible impact (or no impact) they'll have for the resource recreation?

Reply:
Can we add something like "Copilot will create security group if you use udp. nlb sg is a new aws feature that ....... For more, read (the AWS blog post). This allows...., hence a better security posture." in front?

Users may get confused seeing "To use the new Security Group", because they are not the one that decided that "i'm going to use the new sg" 😂

Reply:
I also feel like this NLB security group thing, needs:
a) a warning from the CLI
b) a dedicated section in the blogspot of v1.31 and v1.32

Reply:
Where should the dedicated blogspot section go 🤔

Reply:
I'm thinking after all sections for each features:
```
## Please read if you use `nlb`: NLB security group enhancement

```